lib/server.js

```diff
@@ -1,18 +1,19 @@
 ReactiveTable = {};
 
 ReactiveTable.publish = function (name, collectionOrFunction, selectorOrFunction, settings) {
-    Meteor.publish("reactive-table-" + name, function (publicationId, filters, fields, options, rowsPerPage) {
+    Meteor.publish("reactive-table-" + name, function (publicationId, filters, fields, options, rowsPerPage, serverArgs) {
       check(publicationId, String);
       check(filters, [Match.OneOf(String, Object)]);
       check(fields, [[String]]);
       check(options, {skip: Match.Integer, limit: Match.Integer, sort: Object});
       check(rowsPerPage, Match.Integer);
+      check(serverArgs, Match.Optional(Match.Any));
 
       var collection;
       var selector;
-      
+
       if (_.isFunction(collectionOrFunction)) {
-        collection = collectionOrFunction.call(this);
+        collection = collectionOrFunction.call(this, serverArgs);
```

Comment:
What do you think of making serverArgs an array and using `collectionOrFunction.apply(this, serverArgs)` instead of `call`?

I was trying to make ReactiveTable.publish as similar to Meteor.publish as possible, and I think that would be closer since it allows multiple arguments. It would be `ReactiveTable.publish('name', function (arg1, arg2, arg3) {...})` instead.